app/views/transfer/_welcome_maintenance.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="well maintenance-message">
+  <% if ENV.fetch('MAINTENANCE_MESSAGE_TRANSFER', false) %>
+    <h4 class="hd-5"><%= ENV['MAINTENANCE_MESSAGE_TRANSFER'].to_s %></h4>
```

Comment:
We should be able to trust this input and making this `.html_safe` instead of `.to_s` would allow more flexibility in the custom messages.